meson.build

```diff
@@ -1,12 +1,17 @@
-project('jose', 'c', license: 'APL2', version: '10')
+project('jose', 'c', license: 'APL2', version: '10',
+  default_options: [
+    'c_std=gnu99',
+    'prefix=/usr',
+    'warning_level=2',
+    'werror=true'
+  ]
+)
+
+licensedir = join_paths(get_option('prefix'), 'share', 'licenses', meson.project_name()+'-'+meson.project_version())
```

Comment:
Hm. I am not sure about having the version here. Perhaps stop at the project name?

Reply:
While the standard for Linux seems to be to not have the version in the directory name, the standard for BSD seems to be to have the version in the directory name.  Since the license was not previously included on Linux, I missed difference.  I've updated this to separate out the Linux and BSD styles mimicing the method I used for the tang changes.